certbot/certbot/_internal/storage.py

```diff
@@ -1085,6 +1085,14 @@ def private_key_type(self):
         else:
             return "ECDSA"
 
+    @property
+    def reuse_key(self) -> bool:
```

Comment:
It looks like we currently only access `reuse_key` on `NamespaceConfig` objects, not `RenewableCert` objects. If that's correct, what do you think about deleting this property until we have a use for it?

(The return value of `None` when `reuse_key` isn't in `renewalparams` may be fine, but it honestly makes me a bit nervous at first glance.)

Reply:
Yep, I think this is a leftover from something I was trying but didn't keep.